src/test/java/com/crud/tasks/TasksApplicationTests.java

```diff
@@ -1,13 +1,31 @@
 package com.crud.tasks;
 
+import com.crud.tasks.domain.Task;
+import com.crud.tasks.service.DbService;
+import org.junit.Assert;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 class TasksApplicationTests {
 
+	@Autowired
+	DbService dbService;
+
 	@Test
 	void contextLoads() {
 	}
 
+	@Test
+	public void testGetTaskById(){
+		//Given
+		Long id = 1L;
+		//When
+		Task task = dbService.getTaskById(id);
+		//Then
+		Assert.assertEquals("test",task.getTitle());
+		Assert.assertEquals("test1",task.getContent());
```

Comment:
ten jest jest nieprawidlowy. 
- gdyby to byl test integracyjny testujacy kilka warstw aplikacji, to powinien zaczynac sie od sekcji gdzie zapisujesz element do bazy danych
- to raczej powinien byc test jednostkowy, z mockiem taskRepository (Mockito)